docs/pywbemcli/commands.rst

```diff
@@ -2879,15 +2879,19 @@ implementations of WBEM servers such as the implementation of OpenPegasus.
 
 The ``connection`` command group includes commands that manage named connection
 definitions that are persisted in a :term:`connections file`.
-This allows maintaining multiple connection definitions and then using any
+This allows maintaining multiple connection :term:1definition`s and then using any
 one via the :ref:`--name general option`. Only a single connection is
 active (selected) at any point in time but the connection connection can
 be selected on the pywbemcli command line (:ref:`--name general option`) or
 changed within an interactive session using the :ref:`Connection select command`
 
 .. index:: pair: connections file; persistent connection attributes
 
-The attributes of each connection definition in the :term:`connections file` are:
+TODO: define persistent connection attributes or should this be connection definition
```

Comment:
I guess this is the same. Let's use one consistent term, e.g. "connection definition"

Reply:
Removed the TODO